scripts/link_files.sh

```diff
@@ -1,15 +1,11 @@
 #!/bin/bash
 #
 # A script to link required files to the proper place i.e. where the sample config files will look for them
-# If running on IC machines this will work out of the box
-# If not change FILESDIR to point to where the files live
 
 MACH3DIR=`pwd`
+#FILESDIR=/vols/t2k/users/ljw20/data/DUNE_2021/DUNE_2021_splines_tdr_v8
```

Comment:
This file seems incredibly hardcoded to be used at Imperial - can delete?

Reply:
Left for now - should think about just directly putting this information into the README